core/domain/learner_playlist_services_test.py

```diff
@@ -100,162 +101,203 @@ def _get_all_learner_playlist_collection_ids(self, user_id):
             learner_playlist_model.collection_ids if
             learner_playlist_model else [])
 
-    def test_mark_exploration_to_be_played_later(self):
+    def test_subscribed_exploration_cannot_be_added_to_playlist(self):
+        # Subscribe to exploration.
+        subscription_services.subscribe_to_exploration(
+            self.user_id, self.EXP_ID_0)
+
+        self.assertEqual(
+            self._get_all_learner_playlist_exp_ids(self.user_id), [])
+        # Now if we try to add the same exploration, it shouldn't be added.
+        learner_progress_services.add_exp_to_learner_playlist(
+            self.user_id, self.EXP_ID_0)
         self.assertEqual(
             self._get_all_learner_playlist_exp_ids(self.user_id), [])
 
-        # Add an exploration to the learner playlist.
+    def test_single_exploration_is_added_correctly_to_playlist(self):
+        # Test adding a single exploration_id to learner playlist.
+        self.assertEqual(
```

Comment:
I made a previous comment on this, but can you make these three lines look more like lines 109-110? It is a bit odd to break the first arg across lines when you don't need to.

Reply:
Got it!